modules/ephemeral-storage-csi-inline-overview.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/ephemeral-storage-csi-inline.adoc
+
+[id="ephemeral-storage-csi-inline-overview_{context}"]
+= Overview of CSI inline volumes
+
+Traditionally, volumes that are backed by Container Storage Interface (CSI) drivers can only be used with a PersistentVolume and PersistentVolumeClaim object combination.
+
+This feature allows you to specify CSI volumes directly in the Pod specification, rather than in a PersistentVolume. Inline volumes are ephemeral and do not persist across Pod restarts.
+
+== Support limitations
+
+By default, {product-title} supports CSI inline volumes with these limitations:
+
+* Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
```

Comment:
@bobfuru I suggest adding `ephemeral` because of this line. In this line we said `In-tree and FlexVolumes` does not support inline volume. As I know,  in-tree plug-ins support inline volume, but it's not ephemeral.

Reply:
Ah, I did not know that. Thanks for pointing it out! I've updated throughout the doc to read "CSI inline ephemeral volumes" so that should help emphasize the distinction.